### PR TITLE
Some patches to run it in blender 2.92 (latest for windows 7)

### PR DESCRIPTION
**Pull request by mechanic-kharkiv**
It does not even enables in the add-on tab of preferences dialog without these changes.
After patching, I can run it, and play further. And no scary garbage in the console :-)

### CODE REVIEW

File: ui.py [hunk 1]
```diff
@@ -298,7 +298,10 @@ def execute(self, context):
 
         # change the menu title to the most recently chosen option
         preset_class = VIEW3D_MT_retarget_presets
-        preset_class.bl_label = bpy.path.display_name(basename(filepath), title_case=False)
+        if bpy.app.version < (2, 93, 0):
```
Comment (mechanic-kharkiv) [inline ui.py:301]: users in older versions will see their preset names in title case. otherwise - exception raises;

File: ui.py [hunk 2]
```diff
@@ -424,7 +424,7 @@ class SetToActiveBone(Operator):
     attr_name: StringProperty(default="", options={'SKIP_SAVE'})
     sub_attr_name: StringProperty(default="", options={'SKIP_SAVE'})
     slot_name: StringProperty(default="", options={'SKIP_SAVE'})
-    attr_ptr = PointerProperty(type=properties.RetargetBase)
+    attr_ptr: PointerProperty(type=properties.RetargetBase)
```
Comment (mechanic-kharkiv) [inline ui.py]: This have already been fixed in upstream.

**Comment by mechanic-kharkiv** [comment]
I made it draft because need some time to process upstream updates. Maybe everything is already fixed.

**Comment by pKrime** [comment]
Hi, apologies for taking my time: will look into this ASAP

**Comment by mechanic-kharkiv** [comment]
I've checked updates. They broke some new things, but those, I fixed in this PR are still actual, so it worth to look it over.
The mast mysterious one is `_extra_` dir. It fetches as empty dir, resulting in
`  File "C:\Users\mechanic\AppData\Roaming\Blender Foundation\Blender\2.92\scripts\addons\Expy-Kit-main\__init__.py", line 45, in register
    _extra_.register_classes()
AttributeError: module 'Expy-Kit-main._extra_' has no attribute 'register_classes'`

File: properties.py [hunk 3]
```diff
@@ -145,3 +146,4 @@ def unregister_classes():
 
     bpy.utils.unregister_class(RetargetArm)
     bpy.utils.unregister_class(RetargetLeg)
+    bpy.utils.register_class(RetargetBase)
```
Comment (mechanic-kharkiv) [inline properties.py:149]: Oops. :-/ unregister, of course.

Comment (mechanic-kharkiv) [reply properties.py]: fixed in 7ab759a

File: __init__.py
Comment (mechanic-kharkiv) [inline __init__.py:1]: This _extra_ folder, as I see, is for paid features, and is not cloned. In the `ui.py`  it's handled correctly, so let it be ignored here as well.

**Comment by mechanic-kharkiv** [comment]
Still learning git. So, this looks intimidating to adopt. I will re-arrange some commits, push a new branch to be easier to understand and merge. Then I'll start from an issue, and a new PR.

**Pull request closed without merging**